book/tut4_geophysics.ipynb

```diff
@@ -0,0 +1,1397 @@
+{
```

Comment:
TEST: Maybe introduce a variable at the beginning to have always the same value and be able to change it for all occurrences identically.

Reply:
Done in commit https://github.com/GenericMappingTools/agu24workshop/pull/8/commits/477406eb81203b3e50715f895cdfb9b7c9232349.